src/Domain.LinnApps/Requisitions/RequisitionManager.cs

```diff
@@ -879,6 +890,20 @@ private async Task DoChecksForAutoHeader(RequisitionHeader header)
                     "O"));
             }
 
+            DoProcessResultCheck(this.storesService.ValidStockPool(header.Part, stockPool));
+            
+            // todo - check this isn't going to break anything? 
+            if (header.FromLocation != null || header.FromPalletNumber.HasValue)
+            {
+                DoProcessResultCheck(
+                    await this.stockService.ValidStockLocation(
+                        header.FromLocation?.LocationId, 
+                        header.FromPalletNumber, 
+                        header.Part?.PartNumber,
```

Comment:
need to check if this relies on there being a part number or not as it won't in every function like MOVELOC or SUREQ